easybuild/easyconfigs/g/GEM-library/GEM-library-20121106-022124_pre-release-2_Linux-x86_64.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Daoming Wang
+# Department of Genetics
+# University Medical Center Groningen
+
+easyblock = "Binary"
+
+name = 'GEM-library'
+version = '20121106-022124'
+versionsuffix = '_pre-release-2_Linux-x86_64'
+
+homepage = 'http://algorithms.cnag.cat/wiki/The_GEM_library'
+description = """ Next-generation sequencing platforms (Illumina/Solexa, ABI/SOLiD, etc.) 
+ call for powerful and very optimized tools to index/analyze huge genomes. The GEM library 
+ strives to be a true "next-generation" tool for handling any kind of sequence data, offering 
+ state-of-the-art algorithms and data structures specifically tailored to this demanding task. 
+ At the moment, efficient indexing and searching algorithms based on the Burrows-Wheeler 
+ transform (BWT) have been implemented. The library core is written in C for maximum speed, 
+ with concise interfaces to higher-level programming languages like OCaml and Python. 
+ Many high-performance standalone programs (mapper, splice mapper, etc.) are provided along 
+ with the library; in general, new algorithms and tools can be easily implemented on the top of it. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [('https://sourceforge.net/projects/gemlibrary/files/gem-library/Binary%20pre-release%202/', 'download')]
+sources = ['GEM-binaries-Linux-x86_64-core_2-20121106-022124.tbz2']
```

Comment:
Wouldn't using the `core_i3` tarball make more sense, given the `_Core_i3_is_faster-Use_Core_2_only_on_old_computers_` "warning" in https://sourceforge.net/projects/gemlibrary/files/gem-library/Binary%20pre-release%202/?